packages/twenty-server/src/workspace/workspace-migration-builder/factories/workspace-migration-relation.factory.ts

```diff
@@ -43,11 +43,92 @@ export class WorkspaceMigrationRelationFactory {
           originalObjectMetadataMap,
           relationMetadataCollection,
         );
+      case WorkspaceMigrationBuilderAction.UPDATE:
+        return this.updateRelationMigration(
+          originalObjectMetadataMap,
+          relationMetadataCollection,
+        );
       default:
         return [];
     }
   }
 
+  private async updateRelationMigration(
+    originalObjectMetadataMap: Record<string, ObjectMetadataEntity>,
+    relationMetadataCollection: RelationMetadataEntity[],
+  ): Promise<Partial<WorkspaceMigrationEntity>[]> {
+    const workspaceMigrations: Partial<WorkspaceMigrationEntity>[] = [];
+
+    for (const relationMetadata of relationMetadataCollection) {
+      const toObjectMetadata =
+        originalObjectMetadataMap[relationMetadata.toObjectMetadataId];
+      const fromObjectMetadata =
+        originalObjectMetadataMap[relationMetadata.fromObjectMetadataId];
+
+      if (!toObjectMetadata) {
+        throw new Error(
+          `ObjectMetadata with id ${relationMetadata.toObjectMetadataId} not found`,
+        );
+      }
+
+      if (!fromObjectMetadata) {
+        throw new Error(
+          `ObjectMetadata with id ${relationMetadata.fromObjectMetadataId} not found`,
+        );
+      }
+
+      const toFieldMetadata = toObjectMetadata.fields.find(
+        (field) => field.id === relationMetadata.toFieldMetadataId,
+      );
+
+      if (!toFieldMetadata) {
+        throw new Error(
+          `FieldMetadata with id ${relationMetadata.toFieldMetadataId} not found`,
+        );
+      }
+
+      const migrations: WorkspaceMigrationTableAction[] = [
+        {
+          name: computeObjectTargetTable(toObjectMetadata),
+          action: 'alter',
+          columns: [
+            {
+              action: WorkspaceMigrationColumnActionType.DROP_RELATION,
+              columnName: `${camelCase(toFieldMetadata.name)}Id`,
```

Comment:
is this working with custom fields? I think it should not, we have a helper computeCustomName

Reply:
I'm pretty sure this whole migration thing is skipping custom entities? I've simply copy pasted that part from the create use case in the class below

Reply:
mmh, let me test locally :D

Reply:
Yes please haha